src/themes/ec-core/_index.scss

```diff
@@ -1,7 +1,55 @@
 @use '@ecl/theme-dev/globals' with (
   $border-radius: 'no'
 );
-@use '@ecl/theme-dev/maps/color';
+@use '@ecl/theme-dev/maps/color' with (
+  $color: (
+    'blue': #004494,
```

Comment:
Not so important for the task at hand, but I will comment it as I notice it: the color formats are different, one in RGB and the other in hex.

Reply:
yes, I changed it back to hexa, as it is on ECL 2 (not sure why they got converted to rgb at some point...)